Tests/check_spider_dos.py

```diff
@@ -0,0 +1,8 @@
+# Tests potential DOS of SpiderImagePlugin when seeking in a non-stack file.
+# Run from anywhere that PIL is importable.
+
+from PIL import Image, ImageSequence
+
+im = Image.open("images/hopper.spider")
```

Comment:
Please can you update this to "Tests/images/" so it can be run from the project root? That'll make it consistent with other checks/tests.